Towny/src/main/java/com/palmergames/bukkit/towny/object/Government.java

```diff
@@ -320,6 +322,20 @@ public String getMapColorHexCode() {
 		return mapColorHexCode;
 	}
 
+	/**
+	 * Gets the map color of a nation.
```

Comment:
Not necessarily a Nation, it can be a Town or a Nation, depending on what the Government really is.